File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/commands/RemoveSeatType.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class RemoveSeatType : Command<Guid>
    {
        public Guid SeatTypeId { get; set; }

        public RemoveSeatType() { }
        public RemoveSeatType(Guid conferenceId) : base(conferenceId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/RemoveSeatType.java
package com.microsoft.conference.common.management.commands;

import lombok.Getter;
import lombok.Setter;
import org.enodeframework.commanding.Command;

@Getter
@Setter
public class RemoveSeatType extends Command<String> {
    private String seatTypeId;

    public RemoveSeatType() {
    }

    public RemoveSeatType(String conferenceId) {
        super(conferenceId);
    }
}
